BrowserKit/Tests/.swiftlint.yml

```diff
@@ -0,0 +1,9 @@
+# Swiftlint configuration overloads that will be applied to all files
+# in this folder and all its children recursively, unless another
+# nested configuration takes over.
+# https://github.com/realm/SwiftLint#nested-configurations.
+
+disabled_rules:
```

Comment:
This looks great! Thanks for the work on this issue 🙌 I'll let my colleagues do the final review!